analysed-packages/libpcap/version-1.10.1/libpcap-1.10.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,4828 @@
+LicenseInfoInFile: LicenseRef-MIT
+LicenseInfoInFile: LicenseRef-NTP
+LicenseInfoInFile: LicenseRef-mit-old-style-no-advert
+FileCopyrightText: <text> Copyright 1991 by the Massachusetts Institute of Technology </text>
+
+##File
+
+FileName: libpcap-1.10.1.tar.gz/libpcap-1.10.1.tar/libpcap-1.10.1/missing/strlcat.c
+SPDXID: SPDXRef-item617396
+FileChecksum: SHA1: 59c81882bcaeac1993389f8a589950df8277041d
+FileChecksum: SHA256: f21aa37ef3885751a3df0e0aacb39b9b8b9e848dd7ac2c10dbd79c3aa6998f85
+FileChecksum: MD5: dc817efc8f9a38b545d1c1fbeeab5292
+LicenseConcluded: LicenseRef-ISC
+LicenseInfoInFile: LicenseRef-ISC
+FileCopyrightText: <text> Copyright (c) 1998, 2015 Todd C. Miller <Todd.Miller@courtesan.com> </text>
+
+##File
+
+FileName: libpcap-1.10.1.tar.gz/libpcap-1.10.1.tar/libpcap-1.10.1/config.sub
+SPDXID: SPDXRef-item617535
+FileChecksum: SHA1: 3f4deebf046d0569fb1d9ecb0f1995d213abf21e
+FileChecksum: SHA256: 0c6489c65150773a2a94eebaa794b079e74a403b50b48d5adb69fc6cd14f4810
+FileChecksum: MD5: f31d2995a3a08c75df588c4800738387
+LicenseConcluded: LicenseRef-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-warranty-disclaimer-e0969aba08e91db7e27dad3c475d6933
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-free-unknown
+LicenseInfoInFile: LicenseRef-gpl-3.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception
+FileCopyrightText: <text> Copyright 1992-2021 Free Software Foundation, Inc.
+Copyright 1992-2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: libpcap-1.10.1.tar.gz/libpcap-1.10.1.tar/libpcap-1.10.1/config.guess
+SPDXID: SPDXRef-item617392
+FileChecksum: SHA1: 3ee9d5321d6b97495f5911bf3d8675cf3be84a54
+FileChecksum: SHA256: 7d1e3c79b86de601c3a0457855ab854dffd15163f53c91edac54a7be2e9c931b
+FileChecksum: MD5: c64564b7c7563dae5ebe016f340c9cd9
+LicenseConcluded: LicenseRef-GPL-3.0-or-later-WITH-Autoconf-exception-generic-3.0 AND LicenseRef-warranty-disclaimer-e0969aba08e91db7e27dad3c475d6933
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-free-unknown
+LicenseInfoInFile: LicenseRef-gpl-3.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception
+FileCopyrightText: <text> Copyright 1992-2021 Free Software Foundation, Inc.
+Copyright 1992-2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: libpcap-1.10.1.tar.gz/libpcap-1.10.1.tar/libpcap-1.10.1/ieee80211.h
+SPDXID: SPDXRef-item617470
+FileChecksum: SHA1: fcba459d16fc7d78eda44d154ab1604edb673249
+FileChecksum: SHA256: 73a78659a652322ec5c8d365453cb525c39685af62433f028d89560ef9116249
+FileChecksum: MD5: 216dd726a19adad21c2361c5082beb4d
+LicenseConcluded: LicenseRef-BSD-3-Clause OR LicenseRef-GPL-2.0-only
```

Comment:
The text of the concluded BSD-3-Clause license is not correct the correct one is:
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 1. Redistributions of source code must retain the above copyright
 notice, this list of conditions and the following disclaimer.
 2. Redistributions in binary form must reproduce the above copyright
 notice, this list of conditions and the following disclaimer in the
 documentation and/or other materials provided with the distribution.
 3. The name of the author may not be used to endorse or promote products
 derived from this software without specific prior written permission.

 THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
 IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
 OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
 IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
 INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
 NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
 DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.